### PR TITLE
Add mx ruby_testdownstream_hello for downstream testing in mx

### PR DESCRIPTION
**Pull request by eregon**

### CODE REVIEW

File: mx.truffleruby/mx_truffleruby.py [hunk 1]
```diff
@@ -68,9 +68,14 @@ def ruby_testdownstream(args):
     mx.command_function('build')([])
     mx.run(['ruby', 'tool/jt.rb', 'test', 'fast'])
```
Comment (eregon) [inline mx.truffleruby/mx_truffleruby.py]: We could also make it part of the command above and accept arguments like `specs`, `hello`, etc
@nirvdrum What do you think?

Comment (nirvdrum) [inline mx.truffleruby/mx_truffleruby.py]: I have no real problem with that, but note that if you do so you'll need to ensure some version of MRI is available on the CI machine. Your `ruby_testdownstream_hello` only requires having a JVM and mx available.

File: mx.truffleruby/mx_truffleruby.py [hunk 2]
```diff
@@ -68,9 +68,14 @@ def ruby_testdownstream(args):
...
 
+def ruby_testdownstream_hello(args):
```
Comment (nirvdrum) [inline mx.truffleruby/mx_truffleruby.py]: My gut reaction is this should be called `ruby_testdownstream_basic` or `ruby_testdownstream_spot_check`. But I don't have a huge preference. If you really like `ruby_testdownstream_hello`, that's fine with me.

Comment (eregon) [inline mx.truffleruby/mx_truffleruby.py]: I think being descriptive here helps to be self-explanatory.

File: mx.truffleruby/mx_truffleruby.py (hunk 1 above)
Comment (eregon) [inline mx.truffleruby/mx_truffleruby.py]: Right, I think this way is simpler for now.

**Pull request merged**